### PR TITLE
Update event carousel

### PR DESCRIPTION
**Pull request by mattj-io**
## Description

https://jira.mesosphere.com/browse/DCOS_OSS-3818

## Urgency
- [ ] Blocker 
- [ ] High
- [ x] Medium

## Requirements
- See the [contribution guidelines](https://github.com/dcos/dcos-website#contribution-workflow).
- Build content [locally](https://github.com/dcos/dcos-website#testing-your-updates-locally) and test for formatting/links.

### CODE REVIEW

File: src/events.json [hunk 1]
```diff
@@ -984,18 +990,30 @@
...
 
-  { "title": "Batch to Pipelines",
+  { "title": "Batch to Pipelines (IP Expo Nordics)",
       "image": "/assets/images/events/Stockholm.jpg",
       "date": "2018-09-20",
       "url": "http://www.ipexponordic.com/2018-Speakers/Matt-Jarvis"
   },
 
-  { "title": "Batch to pipelines: Mesos & DC/OS deploy cloud native apps",
+  { "title": "Batch to pipelines: Mesos & DC/OS for managing cloud native apps (Cloud Native London)",
```
Comment (pleia2) [inline src/events.json]: With a title this long, it now runs off the card.

File: src/events.json [hunk 2]
```diff
@@ -960,6 +960,12 @@
       "url": "https://www.meetup.com/DevOpsNorthEast/events/gjhrvlyxjbcc/"
   },
 
+  { "title": "Introduction to DC/OS",
```
Comment (pleia2) [inline src/events.json]: We've typically been putting (Meetup) at the end here so it's clear what it was.

File: src/events.json [hunk 3]
```diff
@@ -984,18 +990,30 @@
...
 
-  { "title": "Batch to pipelines: Mesos & DC/OS deploy cloud native apps",
+  { "title": "Batch to pipelines: Mesos & DC/OS for managing cloud native apps (Cloud Native London)",
       "image": "/assets/images/events/London.jpg",
       "date": "2018-09-26",
       "url": "https://skillsmatter.com/conferences/10160-cloudnative-london-2018#program"
   },
 
+  { "title": "Batch to pipelines: Mesos & DC/OS for managing cloud native apps (IP Expo Europe)",
+      "image": "/assets/images/events/London.jpg",
```
Comment (pleia2) [inline src/events.json]: I'd add a London2.jpg so we don't have identical slides next to each other :smile: We usually get our images from unsplash.com

**Comment by mattj-io** [comment]
@pleia2 - this is ready for another review

**Pull request merged**